lib/search.js

```diff
@@ -6,38 +6,26 @@ var columnify = require('columnify')
 var updateIndex = require('./cache/update-index.js')
 var usage = require('./utils/usage')
 var output = require('./utils/output.js')
+var log = require('npmlog')
+var jsonstream = require('JSONStream')
+var ms = require('mississippi')
 
 search.usage = usage(
   'search',
   'npm search [--long] [search terms ...]'
 )
 
+search.usage = 'npm search [--long] [search terms ...]' +
```

Comment:
This replaces the setting of `search.usage` immediately above it. That seems unnecessary?